### PR TITLE
Fixing social data

### PR DESCRIPTION
**Pull request by jakearchibald**
Just testing…

### CODE REVIEW

File: .github/workflows/preview.yml [hunk 1]
```diff
@@ -7,8 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```
Comment (developit) [inline .github/workflows/preview.yml:11]: See https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit

**Review by developit — APPROVED** [review]

**Comment by jakearchibald** [comment]
I tested this with https://cards-dev.twitter.com/validator and https://developers.facebook.com/tools/debug/.

Twitter is super dumb when it comes to this stuff and I hate it.

File: config.js [hunk 2]
```diff
@@ -3,9 +3,7 @@ export const githubRepository =
   'https://github.com/GoogleChromeLabs/tooling.report/';
 export const githubDefaultBranch = 'live';
 export const githubContribute = `${githubRepository}blob/${githubDefaultBranch}/CONTRIBUTING.md`;
-
-export const metaDescription =
-  'A quick and easy way to figure out what the best tool for your next project is, if it’s worth your time to migrate from one tool to another and how to adopt a best practice into your existing code base. Brought to you by web.dev';
```
Comment (jakearchibald) [inline config.js:8]: This was only used in one page, so I moved it to that page

File: config.js [hunk 3]
```diff
@@ -3,9 +3,7 @@ export const githubRepository =
...
+export const origin = 'https://bundlers.tooling.report';
```
Comment (jakearchibald) [inline config.js:6]: We're only using this in one place right now, but it feels like general config.

File: static-build/components/HeadMeta/index.tsx [hunk 4]
```diff
@@ -11,29 +11,53 @@
...
 
-export default function HeadMeta() {
   return (
     <Fragment>
       <meta
         name="viewport"
         content="width=device-width,initial-scale=1,minimum-scale=1"
       />
+      <title>{title}</title>
+      <meta property="og:title" content={title} />
```
Comment (jakearchibald) [inline static-build/components/HeadMeta/index.tsx:41]: I merged the title into this. Didn't seem to make sense to have them seperate. You'd never have one but not the other.

File: static-build/components/HeadMeta/index.tsx [hunk 5]
```diff
@@ -11,29 +11,53 @@
...
+      <meta name="description" content={description} />
+      {/* Twitter won't use the standard meta description which is stupid but whatever */}
+      <meta property="og:description" content={description} />
       <link rel="shortcut icon" href={icoURL} />
       <link rel="apple-touch-icon" sizes="180x180" href={appleIconURL} />
       <link rel="icon" type="image/png" sizes="32x32" href={png32URL} />
       <link rel="icon" type="image/png" sizes="16x16" href={png16URL} />
       <link rel="mask-icon" href={safariURL} />
       <meta name="color-scheme" content="dark light" />
-      <SocialMeta />
```
Comment (jakearchibald) [inline static-build/components/HeadMeta/index.tsx:36]: This is only used here. Didn't seem to justify being another component.

File: static-build/components/HeadMeta/index.tsx [hunk 6]
```diff
@@ -11,30 +11,56 @@
...
+        property="og:image"
+        content={config.origin + toolingReportSocialURL}
+      />
+      <meta property="og:site_name" content="tooling.report" />
+      <meta name="twitter:site" content="@ChromiumDev" />
+      <meta name="twitter:creator" content="@ChromiumDev" />
+      <meta name="twitter:card" content="summary_large_image" />
+      <meta name="twitter:image:alt" content="tooling.report" />
       <link rel="stylesheet" href={sharedStyles} />
+      <script type="module" async src={analyticsBundleURL}></script>
```
Comment (jakearchibald) [inline static-build/components/HeadMeta/index.tsx:63]: This was included on every page, so I deduped it here.

File: static-build/pages/about/index.tsx [hunk 7]
```diff
@@ -34,14 +32,11 @@ const AboutPage: FunctionalComponent<Props> = () => {
   return (
     <html lang="en">
       <head>
-        <Title parts={['About']} />
-        <meta
-          name="description"
-          content="More direct and ancillary information about this site."
+        <HeadMeta
+          titleParts={['About']}
+          description="More direct and ancillary information about this site."
```
Comment (jakearchibald) [inline static-build/pages/about/index.tsx]:
```suggestion
          description="Information about the tooling.report website - what is tooling.report?"
```

**Review by developit — APPROVED** [review]

**Comment by jakearchibald** [comment]
Taaa. That's on its way to live.

**Pull request merged**